manifests/mlops/kf-users.yaml

```diff
@@ -0,0 +1,35 @@
+name: kubeflow-users
+path: modules/mlops/kubeflow-users
+targetAccount: primary
+targetRegion: us-east-1
+parameters:
+  - name: EksClusterAdminRoleArn
+    valueFrom:
+      moduleMetadata:
+        group: core
+        name: eks
+        key: EksClusterAdminRoleArn
+  - name: EksClusterName
+    valueFrom:
+      moduleMetadata:
+        group: core
+        name: eks
+        key: EksClusterName
+  - name: EksOidcArn
+    valueFrom:
+      moduleMetadata:
+        group: core
+        name: eks
+        key: EksOidcArn
+  - name: EksClusterOpenIdConnectIssuer
+    valueFrom:
+      moduleMetadata:
+        group: core
+        name: eks
+        key: EksClusterOpenIdConnectIssuer
+  # - name: KubeflowUsers
```

Comment:
can you remove it if not needed?

Reply:
I would like to leave in as an example of how to configure.  This module will fail on deployment unless the users are defined (by design).

Reply:
Can't we preconfigure users it with the policies below? 
Also, we should remove the alias.

Reply:
@a13zen see me....you lost me here

Reply:
@a13zen the changes we discussed have been implemented